auacm/angular/src/app/app.module.ts

```diff
@@ -0,0 +1,97 @@
+import { BrowserModule } from '@angular/platform-browser';
```

Comment:
Minor thing: I know golang likes for you to organize imports alphabetically (for readability and such). Maybe do that here (and in declarations, if possible)?